docs/storage/storage.md

```diff
@@ -180,7 +180,7 @@ correct version to request when reading nodes.
 #### Reading Leaves
 
 API requests for leaf data involve a straightforward query by leaf data hash, leaf Merkle hash or
-leaf index followed by formatting and marshalling the data to be returned to the client.
+leaf index followed by formatting and marshaling the data to be returned to the client.
```

Comment:
http://www.dictionary.com/browse/marshaling 

Marshalling is the UK spelling Marshaling is the US. 
I looked at the spelling across the repo, and most of the words had US spellings, so I standardized on that.  Let me know if we'd like to standardize on UK spellings.